libs/core/execution/CMakeLists.txt

```diff
@@ -35,6 +35,7 @@ set(execution_headers
     hpx/execution/detail/sync_launch_policy_dispatch.hpp
     hpx/execution/execution.hpp
     hpx/execution/executor_parameters.hpp
+    hpx/execution/executors/adoptive_static_chunk_size.hpp
```

Comment:
Could you please correct the spelling to `adaptive` (everywhere)?

Reply:
sure ;)